src/tool.cpp

```diff
@@ -220,8 +220,8 @@ DigParams getDigParams(const ItemGroupList &groups,
 	if(time_from_last_punch < tp->full_punch_interval){
 		float f = time_from_last_punch / tp->full_punch_interval;
 		//infostream<<"f="<<f<<std::endl;
-		result_time /= f;
-		result_wear /= f;
+		result_time *= f;
```

Comment:
Not sure but i suspect 'return time' is the expected dig time, which should remain '/=' so that it gets longer.
The change to 'result wear' below seems correct and a bugfix.

Reply:
^ @sofar

Reply:
@sofar